src/open_inwoner/cms/plugins/models/userfeed.py

```diff
@@ -0,0 +1,6 @@
+from cms.models import CMSPlugin
+
+
+class UserFeed(CMSPlugin):
+    # TODO add options
```

Comment:
Is this something that will be done in later PRs?

Reply:
Yea, they mentioned they want to enable/disable and configure stuff so I added the options but then it was not part of this ticket.